frontend/angular.json

```diff
@@ -8,13 +8,6 @@
       "schematics": {
         "@schematics/angular:application": {
           "strict": true
-        },
-        "@schematics/angular:component": {
-          "skipTests": true,
-          "style": "none"
-        },
-        "@schematics/angular:service": {
-          "skipTests": true
```

Comment:
These lines should not be removed, might be a rebase error.

Reply:
Blargh, correct, I'm still getting used to the single commit PR, reverted this file

Reply:
You don't have to use single commit PRs, I'll also merge multi-commit PRs. Do what you think is best for you.